RetailSuiteWarehousing/Integration/03_ConfigureStorageShelves.md

```diff
@@ -0,0 +1,131 @@
+
+The first storage shelf that is created for a material is automatically set as primary. You can subsequently create another storage shelf and set it as primary by clicking the *is primary storage* checkbox. 
+
+Storage shelves can be created, edited, and delete. Bear in mind, though, that you cannot delete a storage shelf where postings have been made in the past, as all posting information is kept in the system. For detailed information, see [Check the posting history](../Operation/03_CheckPostingHistory.md).
+
+
+
+## Create a storage shelf  
+
+You can create an unlimited number of storage shelves in your warehouses to specify exactly where your materials are stored.
+
+#### Prerequisites
+
+- At least one warehouse has been created, see [Create a warehouse](./02_ConfigureWarehouses.md#create-a-warehouse).
+- The warehouse logistics must be active for the selected material, see [Activate the warehouse logistics for a material](../Operation/01_ManageWarehouse.md#activate-the-warehouse-logistics-for-a-material).
+
+#### Procedure
+
+*Warehousing > Quick posting > Tab QUICK POSTING > Select a material > Tab Warehouse/suppliers > Sub-tab Warehouse management*
+
+![Warehouse management](../../Assets/Screenshots/RetailSuiteWarehousing/QuickBooking/WarehouseSuppliers/WarehouseManagement/WarehouseManagement.png "[Warehouse management]")
+
+1. Click the [CREATE] button in the bottom left corner of the *Storage locations* section.  
+    The *Add* section is displayed.
+
+    ![Add storage shelf](../../Assets/Screenshots/RetailSuiteWarehousing/QuickBooking/WarehouseSuppliers/WarehouseManagement/AddStorageShelf.png "[Add storage shelf]")
+
+2. Click the *Warehouse ID* drop-down list and select the warehouse where you want to create the storage shelf.  
+    All available warehouses are displayed in the list.
+
+3. Click the *Storage shelf* field and enter a unique number. This is the storage shelf ID.  
+
+4. If desired, click the *is primary storage shelf* checkbox to set the storage shelf as primary for the selected material.  
+
+    > [Info] The primary storage shelf is then marked with **1** in the **is primary storage shelf** column in the *Storage locations* section.  
+
+5. If you want to create a storage shelf for blocked storage, click the *Blocked storage* drop-down list and select the applicable material conditions. The following options are available:  
+    - **ALL CONDITIONS**  
+    - **NO ORIGINAL PACKAGING**  
+    - **Incomplete**
+    - **Defective item**
+    - **Defective item, to be repaired**
+    - **Out of withdrawal period**
+    - **Out of warranty**
+    - **Used**
+    - **Wrong serial number**
+
+    > [Info] You can create different blocked storage shelves for different material conditions. 
+    
+6. Click the [SAVE] button to save the storage shelf.  
```

Comment:
Oh hier habe ich erst den falschen Save button gedrückt. Es ist der rechts neben "is primary storage"